full/src/main/java/apoc/bolt/Bolt.java

```diff
@@ -23,6 +23,7 @@
 import org.neo4j.driver.types.Node;
 import org.neo4j.driver.types.Path;
 import org.neo4j.driver.types.Relationship;
+import org.neo4j.driver.internal.async.connection.EventLoopGroupFactory;
```

Comment:
Here it looks like you have added an import without adding any code, so I suspect it is unused.

Reply:
Good catch